### DISCUSSION TITLE
How to add new horizontal dividers?

### DISCUSSION THREAD

**Discussion by jodros** [Omikhleia/markdown.sile#136] [Q&A] [answered]
I added this to the function `markdown:internal:thematicbreak`:

```lua
elseif hasClass(options, "loopedsquare") then
    SILE.call("center", {}, function()
    SILE.call("rotate", { angle = 90}, { "⌘ " })
    end)
```

But when I run this in djot, I get a big rule instead!

**Comment by Omikhleia** [comment]
Where did you add that code exactly? It seems to me it should work, when placed at the right position (= before the last "catch all but none" elseif)...

**Reply by Omikhleia** [reply]
Any progress on your side?

**Reply by jodros** [reply]
It started to work, but I don't remember why nor when, sorry.

**Comment by Omikhleia** [comment]
If using Djot with our symbol extensions, also note you might not be obliged to use true semantic breaks and modify the package's code.

You could as well try:
```
[^:mycustombreak:]: ⌘{custom-style="mycustomstyle"}

Lorem...

:mycustombreak:

Lorem...
```

Actually, some clever guy (@ul) noticed and used that [technique](https://github.com/ul/ddd-reference/blob/16f25ca91b64d07066969cf259c7310b5295958a/03-putting-the-model-to-work.dj#L5C1-L5C43).

**Reply by Omikhleia** [reply, marked as answer]
On "new" dividers via a programmatic approach, see also https://github.com/Omikhleia/resilient.sile/discussions/115#discussioncomment-11812882

**Reply by Omikhleia** [reply]
@jodros Would this better answer the question? If so, please "Mark as answer"; if not, let me know what remains to be answered.

**Reply by Omikhleia** [reply]
ping @jodros
